packages/database/prisma/migrations/20241010164405_add_run_unexecuted_setting/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `runUnexecutedblocks` on the `Document` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Document" DROP COLUMN "runUnexecutedblocks",
```

Comment:
this drop column is suspicious, why is it here? did you first added with a typo? i'm afraid this might break